rest/taskana-rest-spring/src/test/java/pro/taskana/common/rest/AccessIdControllerIntTest.java

```diff
@@ -128,6 +139,23 @@ void should_returnAccessIdsOfGroupsTheAccessIdIsMemberOf_ifAccessIdOfUserIsGiven
                 + "cn=Organisationseinheit KSC,cn=organisation,OU=Test,O=TASKANA");
   }
 
+  @Test
+  void should_returnBadRequest_ifAccessIdOfUserContainsInvalidCharacter() {
```

Comment:
Does this test make sense? You want to test something within the LdapClient, but the exception is thrown in the AccessIdController. Thus this test only tests that getGroupsByAccessId is called with a valid accessId

Reply:
This makes sense in case we do a refactoring. In the end, the test makes sure that this case is correctly handled, regardless the way we do it... 
Ok for you?

Reply:
For now, yes :)